scripts/03-post-deploy-mgmt-cluster.sh

```diff
@@ -7,10 +7,34 @@ MANAGEMENT_CLUSTER_NAME=$(yq e .management-cluster.name $PARAMS_YAML)
 WORKER_REPLICAS=$(yq e .management-cluster.worker-replicas $PARAMS_YAML)
 IAAS=$(yq e .iaas $PARAMS_YAML)
 
+
+
+
 tanzu cluster scale $MANAGEMENT_CLUSTER_NAME -n tkg-system -w $WORKER_REPLICAS
 
 kubectl config use-context $MANAGEMENT_CLUSTER_NAME-admin@$MANAGEMENT_CLUSTER_NAME
 
+#Patch the management cluster when Cloud gate session tokens are being used.
+
+if [ -z "$AWS_SESSION_TOKEN" ]; 
```

Comment:
@jeffellin I think we should need to rethink this just a bit.  The script is run for all IAAS options, but the current changes assume AWS.  Can we add at test for .iaas = aws in the original if statement? There should be cases of this in other scripts.  Also, I think we should move the velero and external dns policy changes to the the verlero and extenral dns scripts.  That way if a user doesn't want to run those scripts, the policy updates won't be executed.  Same thing goes in those scrirpts, can't assume aws, so must test for .iaas in the params.yaml file.

Reply:
I can add `elif [ "$IAAS" = "aws" ]; ` but see my comments on other changes before I make that change.